app/assets/stylesheets/application.scss

```diff
@@ -1,14 +1,27 @@
+@import "font-awesome";
 @import "bootstrap-sprockets";
 @import "theme/variables";
 @import "bootstrap";
+@import "daterangepicker-bs3";
 @import "theme/bootswatch";
 
 @import "helpers/colors";
 @import "helpers/glyphicons";
 @import "helpers/fonts";
 @import "helpers/text-aligning";
 @import "helpers/images";
+@import "helpers/buttons";
+@import "helpers/forms";
+@import "helpers/size";
 
+@import "components/nav";
+@import "components/pagination";
+@import "components/daterangepicker";
```

Comment:
Wasn't sure where to put this, but a few request here:

1. Can you remove the shadows from the buttons: https://cl.ly/3M3B3U0C2V1h
2. Can you change the background of the daterange picker to be white: https://cl.ly/26403z1U303b
3. Can you reduce the font size to match: https://cl.ly/26403z1U303b
4. Add some spacing to the top of the dropdown so that it's a few pixels below the input: https://cl.ly/291S1C0U3S16. It should look like: https://cl.ly/2f1B3W0j0X40